Packs/CommonScripts/Scripts/script-GenerateRandomString.yml

```diff
@@ -46,6 +46,9 @@ script: >
 
   characters = ""
 
+  if length > 10000:
+      length = 10000
```

Comment:
You can also use the `max` function.
Please make sure to extract the 10000 magic number to a constant.

Reply:
Done, I used `min` as we want the lower of the two.